src/main/java/org/humancellatlas/ingest/biomaterial/web/BiomaterialController.java

```diff
@@ -77,4 +94,93 @@
+        });
+
+        return ResponseEntity.ok().build();
+    }
+
+    @RequestMapping(path = "/biomaterials/{id}/derivedByProcesses", method = {PUT, POST}, consumes = {TEXT_URI_LIST_VALUE})
+    HttpEntity<?> overrideLinkBiomaterialAsDerivedByProcessesDefaultEndpoint(@PathVariable("id") Biomaterial biomaterial,
+                                                                             @RequestBody Resources<Object> incoming,
+                                                                             HttpMethod requestMethod,
+                                                                             PersistentEntityResourceAssembler assembler) throws URISyntaxException {
+
+        List<Process> processes = uriToEntityConversionService.convertLinks(incoming.getLinks(), Process.class);
+        List<Process> unlinkedProcesses = new ArrayList<>();
+        if (requestMethod.equals(HttpMethod.POST)) {
+            processes.forEach(process -> {
+                biomaterial.addAsDerivedByProcess(process);
+            });
+        } else if (requestMethod.equals(HttpMethod.PUT)) {
+            unlinkedProcesses = new ArrayList(Arrays.asList(biomaterial.getDerivedByProcesses().toArray()));
+            biomaterial.getDerivedByProcesses().clear();
+            biomaterial.getDerivedByProcesses().addAll(processes);
+        }
+
+        biomaterialRepository.save(biomaterial);
+
+        List<MetadataDocument> metadataToSetToDraft = new ArrayList<>();
+        metadataToSetToDraft.addAll(unlinkedProcesses);
+        metadataToSetToDraft.addAll(processes);
+        metadataToSetToDraft.add(biomaterial);
+        metadataToSetToDraft.forEach(metadataDocument -> {
+            validationStateChangeService.changeValidationState(metadataDocument.getType(), metadataDocument.getId(), ValidationState.DRAFT);
+        });
+
+        return ResponseEntity.ok().build();
+    }
+
+
+    @DeleteMapping(path = "/biomaterials/{id}/inputToProcesses/{processId}")
+    HttpEntity<?> unlinkBiomaterialAsInputToProcess(@PathVariable("id") Biomaterial biomaterial,
+                                                    @PathVariable("processId") Process process,
+                                                    PersistentEntityResourceAssembler assembler) {
+        biomaterial.removeAsInputToProcess(process);
+        biomaterialRepository.save(biomaterial);
+
+        validationStateChangeService.changeValidationState(biomaterial.getType(), biomaterial.getId(), ValidationState.DRAFT);
+        validationStateChangeService.changeValidationState(process.getType(), process.getId(), ValidationState.DRAFT);
+
+        return ResponseEntity.noContent().build();
+    }
+
+    @DeleteMapping(path = "/biomaterials/{id}/derivedByProcesses/{processId}")
+    HttpEntity<?> unlinkBiomaterialAsDerivedProcess(@PathVariable("id") Biomaterial biomaterial,
```

Comment:
You can also DRY this together with the previous method: `unlinkBiomaterialAsInputToProcess`